ingest/src/main/java/com/microsoft/azure/kusto/ingest/utils/ContainerWithSas.java

```diff
@@ -32,7 +32,18 @@ public BlobContainerClient getContainer() {
         return container;
     }
 
-    public String getEndpoint() {
-        return container.getBlobContainerUrl() + sas;
+    @Override
+    public String getEndpointWithoutSas() {
```

Comment:
Is there a ToString which does this as well?

Reply:
Why? explicit methods are better

Reply:
Because if you print this class it will print the SAS